src/test/java/org/apache/sling/resourceresolver/impl/MockedResourceResolverImplTest.java

```diff
@@ -399,7 +400,7 @@ private Resource buildResource(String fullpath, Iterable<Resource> children) {
     @SuppressWarnings("unchecked")
     private Resource buildResource(String fullpath, Iterable<Resource> children, ResourceResolver resourceResolver, ResourceProvider<?> provider, String ... properties) {
         Resource resource = Mockito.mock(Resource.class);
-        Mockito.when(resource.getName()).thenReturn(getName(fullpath));
+        Mockito.when(resource.getName()).thenReturn(getResourceName(fullpath));
```

Comment:
ResourceUtil.getName() ?

Reply:
I merge in some changes from master to make it easier to merge and so this is not a change of mine.

Reply:
Kept it as this was not my change